app/models/miq_report/generator.rb

```diff
@@ -81,16 +81,7 @@ def table2class(table)
   end
 
   def get_include_for_find
-    (include_as_hash.presence || invent_includes).deep_merge(include_for_find || {}).presence
-  end
-
-  def invent_includes
-    return {} unless col_order
-    col_order.each_with_object({}) do |col, ret|
-      next unless col.include?(".")
-      *rels, _col = col.split(".")
-      rels.inject(ret) { |h, rel| h[rel.to_sym] ||= {} } unless col =~ /managed\./
-    end
+    include_as_hash(include.presence || invent_report_includes).deep_merge(include_for_find || {}).presence
```

Comment:
My question in https://github.com/ManageIQ/manageiq/pull/18205#issuecomment-439141628 is really asking, can we remove `include.presence` from this section entirely?

Reply:
users tend to copy paste the report templates, and most of the report views have a blank include block (285 blank/344):

```
$ ag 'include:\n' product/views/ | wc -l
     344
$ ag 'include:\n\n' product/views/ | wc -l
     285
```